packages/orbit-components/src/utils/mediaQuery/types.d.ts

```diff
@@ -1,24 +1,7 @@
-// Type definitions for @kiwicom/orbit-components
-// Project: http://github.com/kiwicom/orbit
-import type { Interpolation } from "styled-components";
-
-import type { TOKEN } from ".";
-
 export type Devices =
```

Comment:
Is this file even useful if we have the `mediaQuery/index.ts` above? 🤔

Reply:
No. Moved the export of the type to the index